pokebot.py

```diff
@@ -40,16 +40,13 @@ def _exit() -> NoReturn:
     try:
         if config_general['bot_mode'] != 'manual':
             if GetGameState() == GameState.BATTLE:
-                # Search for the text "What will (Pokémon) do?" in `gDisplayedStringBattle`
-                b_What = EncodeString('What')  # TODO English only
-
-                while ReadSymbol('gDisplayedStringBattle', size=4) != b_What:
+                while GetTask('TASK_HANDLEMONANIMATION').get('data', b'0x00000000')[8] != 0: #define tBattlerId data[4]
```

Comment:
This doesn't work with battle scene set to off 😅

Using this instead, seems to work in all games:
```
while GetTask('TASK_PLAYCRYWHENRELEASEDFROMBALL') == {} and GetTask('SUB_8046AD0') == {}:
    PressButton(['B'])
```

Reply:
Nvm, this frequently gets stuck at high speeds 😑

Reply:
at least for Emerald it works... for all languages..

Reply:
Battle scene: off disables animations, so `TASK_HANDLEMONANIMATION` is never called.

Reply:
ye i am with you on that.. was just looking at the one you proposed

Reply:
Ah right, yeah I'm retrying it - I think I was getting stuck somewhere else (I'm just testing a feature to walk back to home tile if the bot wanders off while spinning to win)